android/electrode-reactnative-bridge/src/main/java/com/walmartlabs/electrode/reactnative/bridge/util/BridgeArguments.java

```diff
@@ -220,11 +220,9 @@ public static List getList(Object obj, @NonNull Class listItemClass) {
         }
 
         List<Object> convertedList = new ArrayList<>();
-        if (obj instanceof Bundle[]) {
-            Bundle[] bundles = (Bundle[]) obj;
-
-            for (Bundle bundle : bundles) {
-                Object item = BridgeArguments.objectFromBundle(bundle, listItemClass);
+        if (obj.getClass().isAssignableFrom(Bundle[].class)) {
+            for (Object bundle : (Object[]) obj) {
```

Comment:
`Bundle[]` instead of `Object[]`?

Reply:
Cannot cast Object[] to Bundle[]

Reply:
In the prior code `Bundle[] bundles = (Bundle[]) obj` is possible , don't see why not possible?

Reply:
Because before the array was validated has to be a Bundle[]. When it's not (like Parcelable[]) then the cast will fail.

Reply:
Cast can only be applied to elements of the array ... the Bundle

Reply:
yes makes sense , though we can catch the `ClassCastException` in that case. 
Please merge it 👍